pkg/hwevent/event.go

```diff
@@ -69,24 +69,21 @@ type Event struct {
 	// DataSchema - A link to the schema that the `Data` attribute adheres to.
 	// +optional
 	DataSchema *types.URI `json:"dataSchema,omitempty"`
-
-	Data *Data `json:"data,omitempty" `
 	// +required
+	Data *Data `json:"data,omitempty" `
 }
 
 // String returns a pretty-printed representation of the Event.
 func (e Event) String() string {
 	b := strings.Builder{}
-	b.WriteString("  id: " + e.ID + "\n")
-	b.WriteString("  type: " + e.Type + "\n")
+	b.WriteString("id: " + e.ID + "\n")
```

Comment:
was there a reason to remove space?, I think  it was added  to prettify in the logs

Reply:
added and tested

```
DEBU[0019] published hw event id: 79cc779f-2587-492a-bcb6-b12e0f35a088
type: HW_EVENT
time: 2021-07-21T21:00:57.019946713Z
data:
  version: v1
  data:
    @odata.context: /redfish/v1/$metadata#Event.Event
    @odata.type: #Event.v1_3_0.Event
    Context: any string is valid
    Id: c8a5433e-de17-11eb-8d4b-f40270e4f7f4
    Name: Event Array
    Events[0]:
      Context: any string is valid
      EventGroupId: 0
      EventId: 2162
      EventTimestamp: 2021-07-06T01:17:12-0400
      EventType: Alert
      MemberId: 79694
      Message: The system inlet temperature is greater than the upper warning threshold.
      MessageId: TMP0120
      Severity: Warning 

```